{
  setUpMocks();
  getProject().getAnt().setProperty(AntTest.FAILURES_OR_ERRORS_PROPERTY,"true");
  try {
    test.executeTests();
    fail();
  }
 catch (  GradleException e) {
    assertThat(e.getMessage(),startsWith("There were failing tests. See the report at"));
  }
}
